### DISCUSSION TITLE
ftp upload to HA using ftp manager

### DISCUSSION THREAD

**Discussion by OM-3kChcBjgFR2COfP** [hassio-addons/addon-ftp#121] [General]
Hey
how can i ftp upload to HA using ftp manager?

what is the path to the HA folder in Debian ( doker ) ?

Thnks!

**Comment by frenck** [comment]
> how can i ftp upload to HA using ftp manager?

I have no idea what "ftp manager" is.

> what is the path to the HA folder in Debian ( doker ) ?

I'm not following that question? I'm not sure what Docker or Debian has to do with it either.

**Comment by vit-um** [comment]
> Hey how can i ftp upload to HA using ftp manager?
> what is the path to the HA folder in Debian ( doker ) ?

`docker ps -a` and find "ghcr.io/hassio-addons/ftp/aarch64:4.7.1" and copy the CONTAINER ID 
`docker exec -it CONTAINER ID /bin/bash`
`cd /ftproot/users/username$` - this is  the path to the HA folder in Debian ( doker )

@frenck Is it possible in the addon configuration to change the ftp directory path to: /usr/share/hassio/media/ outside the docker container?
